app/controllers/concerns/doorkeeper_authentication.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module DoorkeeperAuthentication
```

Comment:
@hoangnguyen92dn  

Should this be renamed to `DoorkeeperAuthenticatable` or `HasDoorkeeperAuthentication`?  💭

Reply:
@Tuubz Following the article shared by Olivier, the `has` should be used in case we want to have a relation between 2 entity

> a has relation between the class where we will include the Concern to and a second entity

So I think we could name 2 concerns as below:

- error_handler -> error_handleable
- doorkeeper_authenticatication -> doorkeeper_authenticatable

💭